rs/sns/governance/canister/governance.did

```diff
@@ -264,6 +264,12 @@ type GetSnsInitializationParametersResponse = record {
   sns_initialization_parameters : text;
 };
 
+type CachedUpgradeSteps = record {
+  upgrade_steps : opt Versions;
+  requested_timestamp_seconds : opt nat64;
+  response_timestamp_seconds : opt nat64;
```

Comment:
why do we need a request and response timestamp here?

Reply:
Request timestamp is used by the lock. Response timestamp is used for auditing (we should document this).

Reply:
Documented the meanings of these fields